pep-0622.rst

```diff
@@ -1131,14 +1129,12 @@ ambiguous with capture patterns. Four other alternatives were considered:
 * Use some implicit rules. For example if a name was defined in the global
   scope, then it refers to a constant, rather than represents a capture pattern::
 
-    FOO = 1
-    value = 0
+    assert "spam" in globals() and "spam" not in locals()
+    assert "side" not in globals()
```

Comment:
Maybe just state the conditions in English? I totally misread these the first one I saw them.